### PR TITLE
Repair size

### PR DESCRIPTION
**Pull request by storybehind**
### Changes
- Gets the size in bytes to repair allocation
- Sample json structure:

```
{
    "upload_size" : 1024,
    "download_size": 2048
}
```

### Fixes
-

### Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/gosdk/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

### Associated PRs (Link as appropriate):
- blobber:
- 0chain:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

File: zboxcore/sdk/repairworker.go [hunk 1]
```diff
@@ -71,6 +73,52 @@ func (r *RepairRequest) processRepair(ctx context.Context, a *Allocation) {
...
+		case fileref.FILE:
+			// this returns repair operations required
+			repairOps := r.repairFile(r.allocation, dir)
+			if repairOps == nil {
+				err = fmt.Errorf("fetch repairOps failed")
+				return rs, err
+			}
+			for _, repairOp := range repairOps {
+				if repairOp.OperationType == constants.FileOperationInsert {
+					rs.UploadSize += uint64(repairOp.Mask.CountOnes()) * uint64(getShardSize(repairOp.FileMeta.ActualSize, r.allocation.DataShards, dir.EncryptionKey != ""))
```
Comment (Hitenjain14) [inline zboxcore/sdk/repairworker.go]: The dir object won't have encrypted key, the file object will have it. You can add a private field to operation request and set if the file is encrypted.

**Review by Hitenjain14 — CHANGES_REQUESTED** [review]

File: zboxcore/sdk/repairworker.go (hunk 1 above)
Comment (storybehind) [reply zboxcore/sdk/repairworker.go]: addressed

**Review by Hitenjain14 — APPROVED** [review]
lgtm

**Review by dabasov — APPROVED** [review]
lgtm

**Pull request merged**